src/signTxCatalystRegistration.c

```diff
@@ -40,24 +47,22 @@ bool signTxCatalystRegistration_isFinished()
 
 void signTxCatalystRegistration_init()
 {
-	{
-		ins_sign_tx_aux_data_context_t* auxDataCtx = &(instructionState.signTxContext.txPartCtx.aux_data_ctx);
-		explicit_bzero(&auxDataCtx->stageContext, SIZEOF(auxDataCtx->stageContext));
-	}
-
-	auxDataHashBuilder_init(auxDataHashBuilder);
+	ins_sign_tx_aux_data_context_t* auxDataCtx = AUX_DATA_CTX;
+	explicit_bzero(&auxDataCtx->stageContext, SIZEOF(auxDataCtx->stageContext));
+	auxDataHashBuilder_init(&auxDataCtx->auxDataHashBuilder);
 
-	subctx->state = STATE_CATALYST_REGISTRATION_VOTING_KEY;
+	accessSubContext()->state = STATE_CATALYST_REGISTRATION_VOTING_KEY;
```

Comment:
Maybe a macro SUB_CTX analogous to the other ones? Or even better one with a more specific name? In other places, too.

Reply:
I don't see the advantage. I use the macros to inject ASSERT code into the function call with usable debug info. The subcontext access happens through these macros already.